Orange/widgets/model/owknn.py

```diff
@@ -0,0 +1,65 @@
+from AnyQt.QtCore import Qt
+
+from Orange.data import Table
+from Orange.modelling import KNNLearner
+from Orange.widgets import gui
+from Orange.widgets.settings import Setting
+from Orange.widgets.utils.owlearnerwidget import OWBaseLearner
+
+
+class OWKNNLearner(OWBaseLearner):
+    name = "K-Nearest Neighbors"
```

Comment:
Not in this PR. This PR merges/move kNN from classification and regression into modelling, and doesn't make any (although potentially necessary) fixes in the kNN itself.

You of all people shouldn't request changing kNN in this PR. :)